entity-framework/core/what-is-new/ef-core-3.0/features.md

```diff
@@ -45,6 +45,31 @@ The provider will enable most EF Core features, like automatic change tracking,
 We started this effort before EF Core 2.2, and [we have made some preview versions of the provider available](https://blogs.msdn.microsoft.com/dotnet/2018/10/17/announcing-entity-framework-core-2-2-preview-3/).
 The new plan is to continue developing the provider alongside EF Core 3.0. 
 
+## Dependent entities sharing the table with the principal are now optional
```

Comment:
You are adding this already at the beginning.

Reply:
This is a different page